zeek/compiler/glue-compiler.cc

```diff
@@ -235,6 +235,73 @@ GlueCompiler::GlueCompiler(Driver* driver) : _driver(driver) {}
 
 GlueCompiler::~GlueCompiler() {}
 
+hilti::Result<std::string> GlueCompiler::getNextEvtBlock(std::istream& in, int* lineno) {
+    std::string chunk;
+
+    // Parser need to track whether we are inside a string or a comment.
+    enum State { Default, InComment, InString } state = Default;
+    char prev = '\0';
+
+    while ( true ) {
+        char cur;
+        in.get(cur);
+        if ( in.eof() ) {
```

Comment:
nit: can write this shorter

```suggestion
        if ( in.get(cur) ) {
```

Reply:
Turns out this doesn't work, reverted.